subprojects/core/src/main/java/org/gradle/execution/plan/OrdinalNodeAccess.java

```diff
@@ -80,7 +84,7 @@ private OrdinalNode createOrdinalNode(OrdinalNode.Type type, OrdinalGroup ordina
     }
 
     public OrdinalGroup group(int ordinal) {
```

Comment:
I am not sure it makes sense to expose this method any more. Clients should use `ordinalGroups.group` directly.

Reply:
That would force the execution plan to keep a reference to the factory. I don't see much harm in keeping the delegation here for the smaller change to the execution plan implementation.